### PR TITLE
Update amounts in all apis

### PR DESCRIPTION
**Pull request by dobras**
https://mews.myjetbrains.com/youtrack/issue/TDD-1473

### CODE REVIEW

**Comment by dobras** [comment]
I will ask @skurik  or @ondrahermanek

File: distributor-api-v1/operations.md [hunk 1]
```diff
@@ -104,6 +104,28 @@ Preferred initial call used to obtain all static data about distributor configur
                         "Prices": {
```
Comment (siroky) [inline distributor-api-v1/operations.md]: Why do we still keep the old currencyvalues?

File: distributor-api-v1/operations.md [hunk 2]
```diff
@@ -382,6 +426,7 @@ If the hotel does not use any payment gateway, the value is null. If it does, th
 | `IncludedByDefault` | boolean | required | Indicates whether the product should be added to order by default. |
 | `AlwaysIncluded` | boolean | required | Indicates whether the product is always included \(= cannot be removed\). |
 | `Prices` | [CurrencyValues](operations.md#currencyvalues) | required | Price of the product. |
```
Comment (siroky) [inline distributor-api-v1/operations.md]: Isn't this obsolete?

File: distributor-api-v1/operations.md [hunk 3]
```diff
@@ -588,6 +637,8 @@ Gives availabilities and pricings for given date interval with product prices in
 | :--- | :--- | :--- | :--- |
 | `Total` | [CurrencyValues](operations.md#currencyvalues) | required | Total price of the room for whole reservation. |
```
Comment (siroky) [inline distributor-api-v1/operations.md]: Aren't these two obsolete?

**Pull request merged**